packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/preview/index.ts

```diff
@@ -38,9 +49,26 @@ export default class CardPayPrepaidCardWorkflowPreviewComponent extends Componen
       .catch((e) => console.error(e));
   }
 
+  @action retryIssuePrepaidCard() {
+    taskFor(this.issueTask).cancelAll();
```

Comment:
We should also clear the transaction hash so we don't get the previous transaction's blockscout link when retrying, prior to confirmation in Card Wallet.

Reply:
You aren't allowed to cancel once you have a transaction hash, because at that point things are out of our control and in the miners' hands.